### PR TITLE
Fix: Redact password from URL

### PR DESCRIPTION
**Pull request by xBlaz3kx**
According to #190, password gets logged when connection fails. Hopefully this solves the issue :)

### CODE REVIEW

**Comment by wagslane** [comment]
Thanks so much - honestly though, I think we should just not log the URL at all - would you mind updating your PR to just remove it and I'll get this merged?

**Comment by xBlaz3kx** [comment]
I would keep it though, I think it would be useful for debugging. Perhaps the user specified multiple server connections, each with different u/p or addresses. In case of failure, it would be easier to pinpoint the problematic connection or URL.

File: internal/connectionmanager/connection_manager.go [hunk 1]
```diff
@@ -42,12 +43,19 @@ func dial(log logger.Logger, resolver Resolver, conf amqp.Config) (*amqp.Connect
 		if err == nil {
 			return conn, err
 		}
-		log.Warnf("failed to connect to amqp server %s: %v", url, err)
+
+		// Log masked url
```
Comment (wagslane) [inline internal/connectionmanager/connection_manager.go:47]: rm this comment please

**Review by wagslane — CHANGES_REQUESTED** [review]
alright, im down to leave the url if it's redacted - just one nit

**Pull request merged**